plugins/clickhouse/CMakeLists.txt

```
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.19...3.28 FATAL_ERROR)
+
+project(
+  clickhouse
+  DESCRIPTION "Clickhouse Integration for Tenzir"
+  LANGUAGES CXX)
+
+find_package(Tenzir REQUIRED)
+
+set(WITH_OPENSSL ON CACHE BOOL "")
+set(WITH_SYSTEM_ABSEIL ON CACHE BOOL "")
+set(WITH_SYSTEM_CITYHASH OFF CACHE BOOL "")
+set(WITH_SYSTEM_LZ4 ON CACHE BOOL "")
+set(WITH_SYSTEM_ZSTD ON CACHE BOOL "")
+
+add_subdirectory(aux/clickhouse-cpp)
+TenzirSystemizeTarget(clickhouse-cpp-lib)
+
+TenzirRegisterPlugin(
+  TARGET clickhouse
+  ENTRYPOINT builtins/stub.cpp
+  SOURCES GLOB src/*.cpp
+  BUILTINS builtins/to_clickhouse.cpp
```

```suggestion
  ENTRYPOINT src/plugin.cpp
  SOURCES GLOB src/*.cpp
  BUILTINS builtins/*.cpp
```

Please don't set this up different from other plugins. That'd just add confusion. I would never expect the plugin entry point to be in the builtins directory.